blueprints/cloud-operations/network-dashboard/deploy-cloud-function/main.tf

```diff
@@ -47,7 +47,7 @@ module "pubsub" {
   project_id    = module.project.project_id
   name          = var.name
   regions       = [var.region]
-  subscriptions = { "${var.name}-default" = null }
+  subscriptions = {}
 }
```

Comment:
why this change?

Reply:
I think the subscription was not being used, and the `=null` part doesn't work since I updated the module

Reply:
What Julio said, I think the new version of the module works better with {} (uses the default subscription)

Reply:
Aha Fabric is becoming too large to keep in a single head. Thanks for the clarification. 😀